inst/@sym/sort.m

```diff
@@ -0,0 +1,82 @@
+%% Copyright (C) 2014, 2015 Utkarsh Gautam
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{g} =} sort (@var{f})
+%% Sorting symbols .
+%% Author: Utkarsh Gautam
+%% Keywords: symbolic, sort
+
+function [s,i] = sort(f)
+	if isempty(f)
+		[s,i]=sort(f)
+	else 
+	  if isallconstant(f) 
+	    [s,i]=sort(double(f));
+	    s=sym(s);
+	  else
+	    error('Invalid Input')
+	  end
+	end    
+end
+
+
+%!test 
+%! f = [sym(1),sym(0)] ;
+%! sort(f)	;
+%! expected = sym([0 , 1]) ;
+%! assert (isequal(sort(f), expected));
+
+%!test 
+%! f = [sym(1)] ;
+%! sort(f)	;
```

Comment:
no spaces before semicolon.  Its customary to copy the style of other functions when contributing to a Free Software project.